test/parallel/test-stringbytes-external.js

```diff
@@ -129,31 +129,31 @@ var PRE_3OF4_APEX = Math.ceil((EXTERN_APEX / 4) * 3) - RADIOS;
 
   assert.throws(function() {
     buf1.toString();
-  }, /toString failed|Invalid array buffer length/);
+  }, /toString\(\) failed: buffer larger|Invalid array buffer length/);
```

Comment:
I think you're right. Maybe go ahead and remove the extra message, add the type, and we'll run it through CI and see if anything blows up on it or not.